### PR TITLE
changed hometrainee method, added methods to features

### PR DESCRIPTION
**Pull request by ahmeedo98**
changed method in hometrainee pom, added stepdefs to homeTrainee

### CODE REVIEW

**Review by Danielall2 — APPROVED** [review]

File: src/test/java/com/sparta/greg/cucumber/stepDefs/HomeTraineeStepdefs.java [hunk 1]
```diff
@@ -1,100 +1,167 @@
...
+        traineeUsername = PropertyLoader.properties.getProperty("traineeUsername");
+        traineePassword = PropertyLoader.properties.getProperty("traineePassword");
+
+        Login login = new Login(webDriver);
+        login.logInAsTrainer(traineeUsername, traineePassword);
+    }
+
     @Given("I am logged in as a trainee and on the Home Trainee Page")
     public void iAmLoggedInAsATraineeAndOnTheHomeTraineePage() {
+        loadPropertiesLoginAsTrainerGoToAssessments();
```
Comment (GSpratt97) [inline src/test/java/com/sparta/greg/cucumber/stepDefs/HomeTraineeStepdefs.java:46]: Trainee?

File: src/test/java/com/sparta/greg/cucumber/stepDefs/HomeTraineeStepdefs.java [hunk 2]
```diff
@@ -1,100 +1,167 @@
...
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Properties;
 
 public class HomeTraineeStepdefs {
+
+    private PropertyLoader propertyLoader = new PropertyLoader();
+
+    private static WebDriver webDriver;
+    private static Properties properties = new Properties();
```
Comment (GSpratt97) [inline src/test/java/com/sparta/greg/cucumber/stepDefs/HomeTraineeStepdefs.java:25]: Delete this

File: src/test/java/com/sparta/greg/cucumber/stepDefs/HomeTraineeStepdefs.java [hunk 3]
```diff
@@ -1,100 +1,167 @@
...
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Properties;
 
 public class HomeTraineeStepdefs {
+
+    private PropertyLoader propertyLoader = new PropertyLoader();
```
Comment (GSpratt97) [inline src/test/java/com/sparta/greg/cucumber/stepDefs/HomeTraineeStepdefs.java:22]: Is this needed?

File: src/test/resources/features/HomeTrainee.feature [hunk 4]
```diff
@@ -1,64 +1,70 @@
...
     Then I am taken to the Home Trainee Page from the Home Trainee Page
 
-  Scenario: Trainee goes to Home Trainee Page from the Home Trainee Page (Profile)
+  Scenario: Trainee goes to the Home Trainee Page from the Home Trainee Page (Profile)
     Given I am logged in as a trainee and on the Home Trainee Page
     When I click on Profile on the Home Trainee Page
     Then I am taken to the Home Trainee Page from the Home Trainee Page
 
-  Scenario: Trainee goes to Consultancy Skills Page from Home Trainee Page
+  Scenario: Trainee goes to the Consultancy Skills Page from Home Trainee Page
```
Comment (GSpratt97) [inline src/test/resources/features/HomeTrainee.feature:42]: Wheres the "the"

File: src/test/resources/features/HomeTrainee.feature [hunk 5]
```diff
@@ -1,64 +1,70 @@
...
     Given I am logged in as a trainee and on the Home Trainee Page
     When I click View on the trainee sidebar on the Home Trainee Page
     And I click on Consultancy Skills on the Home Trainee Page
     Then I am taken to the Consultancy Skills Page from the Home Trainee Page
 
-  Scenario: Trainee goes to Trainee Guide Page from Home Trainee Page
+  Scenario: Trainee goes to the Trainee Guide Page from Home Trainee Page
```
Comment (GSpratt97) [inline src/test/resources/features/HomeTrainee.feature:48]: And the "the" here?

**Review by alexkng97 — APPROVED** [review]

**Review by GSpratt97 — APPROVED** [review]

**Pull request merged**